src/model/UserphraseModel.h

```diff
@@ -71,4 +72,5 @@ public slots:
 
     std::unique_ptr<ChewingContext, void (*)(ChewingContext*)> ctx_;
     UserphraseSet userphrase_;
+    int addResult;
```

Comment:
Sorry, it was my fault, I should reply the code review from @Chocobo1 after I modify my code.
I initialize the variable in the initialization list of UserphraseModel, because I want to keep the same coding style.